src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-math-using-select.mdx

```diff
@@ -85,40 +96,33 @@ These three functions force decimal numbers to one of the neighboring integers.
   Sample graph showing raw data, with floor, round, and ceiling functions applied.
 </figcaption>
 
-### `clamp_max`, `clamp_min`
-
-The clamping functions impose an upper or lower bound on values.
-
-For example, `clamp_max(duration, 10)` returns the duration, unless it exceeds 10, in which case 10 is returned.
-
-Similarly `clamp_min(duration, 1)` will not return any value lower than 1. The following chart shows the result of clamping both min and max to keep the value in the range 70-90.
-
-![clamp.png](./images/clamp.png "clamp.png")
+### `sqrt` [#sqrt]
 
-<figcaption>
-  Sample graph showing raw data with clamp function applied.
-</figcaption>
+`sqrt(n)` returns the square root of _n_, that is, the number such that `sqrt(n) * sqrt(n) = n.`
 
-### `pow`
+## Results with STRING or FLOAT [#strings]
 
-`pow(n, m)` computes _n_ raised to the power _m_. (I.e. `n * n * ... * n,` with _m_ copies of _n_)
+Here is how NRQL handles strings present in math calculations:
 
-### `sqrt`
+Examples:
 
-`sqrt(n)` returns the square root of _n_, that is, the number such that `sqrt(n) * sqrt(n) = n.`
+* `sum(1+STRING)` = 0
+* `sum(1+MIXED)` = skips records where MIXED is a string
+* `average(1+STRING)` = 0
+* `average(1+MIXED)` = skips records where MIXED is a string
 
-### `exp`
+NULL and zero both appear as 0 in the dashboard.
 
-Computes the natural exponential function of the argument: `exp(n) = pow(e, n)`.
+To override NULL values with another numeric value, use the syntax:
 
-### `ln, log2, log10, log`
+```
+SELECT average(purchasePrice OR 0)
+```
 
-These functions compute the logarithm of the argument for various bases.
+This will replace NULL values with 0 or any number specified.
 
-* `ln(n)` computes the natural logarithm: the logarithm base e.
-* `log2(n)` computes the logarithm base 2.
-* `log10(n)` computes the logarithm base 10.
-* `log(n, b)` allows logarithms to be computed with an arbitrary base b.
-* All logarithms satisfy the identity: `log(pow(b, n), b) = n.`
+<Callout variant="tip">
+  This can also be used to test whether something returns NULL or zero. `(zero) OR 1` returns 0. `(NULL) OR 1` returns 1.
+</Callout>
 
-Note that `log(0`) is undefined, for all bases. Be aware that if you take the logarithm of something that might be zero, you may end up getting `No Value` back from your query.
\ No newline at end of file
+## For more help [#more-help]
```

Comment:
I don't think you need to add "## For more help". Pretty sure it's added automatically. Right now, it's just showing the header without anything beneath it.